packages/backend/src/entity/controller.ts

```diff
@@ -0,0 +1,66 @@
+import { RouteConfig } from '@asteasolutions/zod-to-openapi'
+import { z } from 'zod'
+import { Entity, Schema } from '@ssi-trust-registry/common'
+import { Request, Response } from 'express'
+import partial from 'lodash.partial'
+import { EntityService } from './service'
+import { SchemaService } from '../schema/service'
+import { createLogger } from '../logger'
+import { RequestWithToken } from '../auth/middleware'
+
+const logger = createLogger(__filename)
+
+const RegistryResponseSchema = z.object({
+  entities: z.array(Entity),
+  schemas: z.array(Schema),
+})
+
+export function createEntityController(
+  entityService: EntityService,
+  schemaService: SchemaService,
+): EntityController {
+  return {
+    getRouteConfigs: partial(getRouteConfigs),
+    getRegistry: partial(getRegistry, entityService, schemaService),
+  }
+}
+
+export interface EntityController {
+  getRouteConfigs: () => RouteConfig[]
+  getRegistry: (req: RequestWithToken, res: Response) => Promise<void>
+}
+
+async function getRegistry(
+  entityService: EntityService,
+  schemaService: SchemaService,
+  req: Request,
+  res: Response,
+) {
+  logger.info('Reading the registry from the file.')
+  const registry = {
+    entities: await entityService.getAllEntities(),
+    schemas: await schemaService.getAllSchemas(),
+  }
+  res.status(200).json(registry)
+}
+
+function getRouteConfigs(): RouteConfig[] {
```

Comment:
I realized that my idea how to name this might be a bit misleading because this object is really used only for documentation purposes.
```suggestion
function getRouteConfigDocs(): RouteConfig[] {
```

Reply:
Or add JS doc
```js
// Beware that returned objects are actually used only for documentation purposes and they're not use for routes definitions.
```

Reply:
I like renaming more than adding docs everywhere.